rdg_datasets/five_node_star_graph/import.py

```diff
@@ -0,0 +1,30 @@
+import os
+import pathlib
```

Comment:
Same comment as above.  These two files shared 90% of code, so we might already have a simpler way to provide the paths, etc for a new graph.